spec/rails_helper.rb

```diff
@@ -32,6 +32,7 @@
 require 'super_diff/rspec-rails'
 require 'super_diff/active_support'
 require './spec/support/default_configuration_helper'
+require 'logging/third_party_transaction'
```

Comment:
(Non blocking) Nitpicky but feel like I've been called out in the past for requiring modules for specific types of tests in global configuration files like `rails_helper`, as too many of these may slow down the test suite overall. I might consider requiring these somewhere else that is less global for the rest of the test suite

Reply:
done